build/templates/UmbracoProject/UmbracoProject.csproj

```diff
@@ -10,47 +13,24 @@
         <PackageReference Include="Umbraco.SqlServerCE" Version="4.0.0.1" Condition="'$(UseSqlCe)' == 'true'" />
     </ItemGroup>
 
-    <Import Project="..\PackageTestSiteName\build\PackageTestSiteName.targets" Condition="'$(PackageTestSiteName)' != ''"/>
+    <Import Project="..\PackageTestSiteName\build\PackageTestSiteName.targets" Condition="'$(PackageTestSiteName)' != ''" />
+
     <ItemGroup Condition="'$(PackageTestSiteName)' != ''">
-        <ProjectReference Include="..\PackageTestSiteName\PackageTestSiteName.csproj"/>
+        <ProjectReference Include="..\PackageTestSiteName\PackageTestSiteName.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-        <Content Remove="umbraco\Data\**" />
-        <Content Remove="umbraco\logs\**" />
-        <Content Remove="umbraco\MediaCache\**" />
-    </ItemGroup>
-    <ItemGroup>
-        <Compile Remove="umbraco\Data\**" />
-        <Compile Remove="umbraco\logs\**" />
-        <Compile Remove="umbraco\MediaCache\**" />
-    </ItemGroup>
-    <ItemGroup>
-        <EmbeddedResource Remove="umbraco\Data\**" />
-        <EmbeddedResource Remove="umbraco\logs\**" />
-        <EmbeddedResource Remove="umbraco\MediaCache\**" />
-    </ItemGroup>
-    <ItemGroup>
-        <None Include="config\**\*.*">
-            <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-            <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-        </None>
-        <None Include="umbraco\**\*.*">
-            <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-            <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-        </None>
-        <None Include="App_Plugins\**\*.*">
-            <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-            <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-        </None>
-        <None Remove="umbraco\Data\**" />
-        <None Remove="umbraco\logs\**" />
-        <None Remove="umbraco\MediaCache\**" />
+        <Content Include="App_Plugins/**/*.*" CopyToOutputDirectory="Always" />
+        <Content Include="umbraco/**/*.*" CopyToOutputDirectory="Always" />
+        <Content Remove="umbraco/Data/**" />
```

Comment:
Assumes deploy files moving from umbraco/Data -> umbraco/Deploy

Reply:
This has been done now (in `v9/dev`).